plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/ColumnWithAliasFormatter.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.jdbc;
+
+import io.trino.spi.connector.ConnectorSession;
+
+import static com.google.common.base.Splitter.fixedLength;
+import static com.google.common.base.Strings.padStart;
+
+public class ColumnWithAliasFormatter
```

Comment:
Maybe rename to `SyntheticColumnHandleProvider/Builder/etc.` and rename the method to `getSyntheticColumnHandle`? What do you think (WDYT)?

Reply:
I'm fine with the names your proposed, since what I don't like what I proposed :)

I usually try to use the DRY (Don't Repeat Yourself) principle in these instances, that is if I use the description of what's going on in the class, then I don't repeat it in the method, but just call the method 'execute' or 'apply'. Otherwise you have a code that looks like:

```
private SynthenticColumnHandleBuilder synthenticColumnHandleBuilder;

...

syntheticColumnHandleBuilder.getSynthenticColumnHandle();
```

On the contrary if you use the apply/execute method the call line reads better IMHO

```
synthenticColumnHandleBuilder.execute()
```

But I don't have a specific preference here, so if you have hard opinion on these names, I can apply them like you proposed. 

Provider has a specific meaning in Guice, and Builder is usually a builder pattern. Both of which might be misleading. If I had to choose only between the two alternatives, I'd probably use Builder. But others that come to my head: Converter, Synthesizer?

Let me know with the variant you choose. I'm OK with any.

Reply:
Just `get` or `getFromColumnHandle` would make sense.